bot.py

```diff
@@ -63,6 +67,55 @@ def get_token():
 		return "".join(f.readlines()).strip()
 
 
+@bot.command()
+async def minigame(ctx: commands.Context): # noqa: D103
+	"""Minigame to allow users to acquire Pokemon."""
+	await lock.acquire()
+
+	await bot.change_presence(activity=discord.Game("Who's That Pokemon?"))
```

Comment:
This is global, and probably won't make that much sense if there isn't a minigame happening. Would prefer if it was removed.